dashboard/lti_new.py

```diff
@@ -234,6 +235,14 @@ def extract_launch_variables_for_tool_use(request, message_launch):
         password = ''.join(random.sample(string.ascii_letters, settings.RANDOM_PASSWORD_DEFAULT_LENGTH))
         user_obj = User.objects.create_user(username=username, email=email, password=password, first_name=first_name,
                                             last_name=last_name)
+
+    # Add username into the MyLA User table, since the data was not pulled in from cron job
+    user_id = CANVAS_DATA_ID_INCREMENT + int(canvas_user_id)
+    try:
+        MyLAUser.objects.filter(user_id=user_id).update(sis_name = username)
```

Comment:
```suggestion
        MyLAUser.objects.filter(user_id=user_id).update(sis_name=username)
```

Reply:
this line is replaced by the following suggested code block:

`enrollment_qs =  MyLAUser.objects.filter(user_id=user_id)
if enrollment_qs.exists():
    enrollment_qs.update(sis_name=username)`